rosdep/python.yaml

```
@@ -6619,6 +6617,7 @@ python3-google-cloud-storage-pip: *migrate_eol_2025_04_30_python3_google_cloud_s
 python3-google-cloud-texttospeech-pip: *migrate_eol_2025_04_30_python3_google_cloud_texttospeech_pip
 python3-googleapi:
   debian: [python3-googleapi]
+  gentoo: [dev-python/google-api-python-client]
   ubuntu: [python3-googleapi]
```

```suggestion
python3-googleapi:
  debian: [python3-googleapi]
  fedora: [python3-google-api-client]
  gentoo: [dev-python/google-api-python-client]
  ubuntu: [python3-googleapi]
```

After looking at why the Fedora package was failing I found out that the while google-api-python-client is the name of the source package, the generated noarch binary package uses the conventional python3- prefixed name. 
Here's an example from Fedora 36 https://koji.fedoraproject.org/koji/buildinfo?buildID=1864742

---

Thanks for the help, was struggling to figure out why that package name wasn't working. Checked what you showed here, and confirmed I was able to install. 

> [root@0e194f62ae53 /]# yum install python3-google-api-client
Fedora 35 - x86_64                                                                                                            5.9 MB/s |  79 MB     00:13     Fedora 35 openh264 (From Cisco) - x86_64                                                                                      1.3 kB/s | 2.5 kB     00:01    
Fedora Modular 35 - x86_64                                                                                                    1.3 MB/s | 3.3 MB     00:02    
Fedora 35 - x86_64 - Updates                                                                                                  3.4 MB/s |  20 MB     00:05    
Fedora Modular 35 - x86_64 - Updates                                                                                          1.2 MB/s | 2.8 MB     00:02    
Dependencies resolved.
==============================================================================================================================================================
 Package                                               Architecture                Version                                 Repository                    Size
==============================================================================================================================================================
Installing:
 python3-google-api-client                             noarch                      2:2.32.0-1.fc35                         updates                      2.6 M